bundler/doc/contributing/HOW_YOU_CAN_HELP.md

```diff
@@ -6,22 +6,22 @@ If at any point you get stuck, here's how to [get in touch with the Bundler team
 
 ## First contribution suggestions
 
-We track [small bugs and features](https://github.com/rubygems/bundler/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) so that anyone who wants to help can start with something that's not too overwhelming.
+We track [small bugs and features](https://github.com/rubygems/rubygems/contribute) so that anyone who wants to help can start with something that's not too overwhelming.
```

Comment:
We do have the "good first issue" label, so I think it's good to keep it? Actually, I plan to do a round of triaging to tag more issues like that because I got potential first time contributors asking me for this kind of issues recently :)

Reply:
so the link should reference [here](https://github.com/rubygems/rubygems/issues?q=is%3Aissue+is%3Aopen+label%3Abundler++label%3A%22good+first+issue%22+), right?

Reply:
Yes!